src/uct/api/v2/uct_v2.h

```diff
@@ -690,21 +715,26 @@ ucs_status_t uct_md_query_v2(uct_md_h md, uct_md_attr_v2_t *md_attr);
 /**
  * @ingroup UCT_MD
  *
- * @brief Pack a remote key.
+ * @brief Pack a memory key as a remote or shared one.
+ *
+ * This routine packs a local memory handle registered by @ref uct_md_mem_reg
+ * into a memory buffer, which then could be deserialized by a peer and used in
+ * UCT operations.
  *
  * @param [in]  md          Handle to memory domain.
  * @param [in]  memh        Pack a remote key for this memory handle.
  * @param [in]  params      Operation parameters, see @ref
  *                          uct_md_mkey_pack_params_t.
- * @param [out] rkey_buffer Pointer to a buffer to hold the packed remote key.
- *                          The size of this buffer has should be at least
- *                          @ref uct_md_attr_t::rkey_packed_size, as returned by
- *                          @ref uct_md_query.
+ * @param [out] mkey_buffer Pointer to a buffer to hold the packed remote key.
```

Comment:
remote key -> memory key

Reply:
fixed in https://github.com/openucx/ucx/pull/8538